app/models/pano/pano_form_builder.rb

```diff
@@ -9,5 +9,9 @@ def formatted_date_field(name, options = {})
     def sanitize_to_id(name)
       name.to_s.delete("]").tr("^-a-zA-Z0-9:.", "_")
     end
+
```

Comment:
You can move this above sanitize_to_id, which should actually remain at the bottom as a private method. (I'll fix that if its not marked as private currently)